docs/packages.md

```diff
@@ -0,0 +1,103 @@
+# DPI Packages
+
+Ruter publishes a packages manifest that provides a list of packages that should be installed on vehicles at the following locations.
+
+| Environment | URL | Purpose |
+| --- | ---| --- |
+| Prod | https://bus-dpi.transhub.io/prod/packages.json | All vehicle running regular routes |
+| Stage | https://bus-dpi.transhub.io/stage/packages.json | Test rigs, vehicles being tested before running regular routes |
+
+These locations should be checked once daily, at a minimum, after 16.00 CET, which is our cutoff for changes for the day, Monday to Friday.
+
+It is expected that new packages should be installed on all vehicles running in regular traffic after 05.00 CET the following day.
+
+## Contents of Manifest
+
+```json
+{
+  "timestamp" : "2021-02-25T14:43:14Z",
+  "packages" : [ {
+    "destination" : "application",
+    "url" : "https://bus-dpi.transhub.io/packages/application-2021-02-25T14-25-12Z.zip",
+    "sha256" : "5258f1e757ab9fb1e222601c793030219b7064b21206d234fc094bbdaf546fd0"
+  }, {
+    "destination" : "application/media",
+    "url" : "https://bus-dpi.transhub.io/packages/media-2019-10-16T15-11-00Z.zip",
+    "sha256" : "44a143c13ab452b7bd5d06f4ce3373af7b5a4da0095b42a3db27bd04e172072b"
+  } ]
+}
+```
+
+When a new version of the packages is published the timestamp will increment.
+
+Use the URL to fetch the packages and check that the downloaded Zip file has the same SHA256 values as in the manifest.
+
+## Placement of Files
+
+Currently the two zip files are organized internally slightly differently, and this affects how they need to be handled. The destination in the manifest refers to where the working directory must be when the file is unzipped.
+
+The packages should end up deployed in a structure like this:
+
+* application
```

Comment:
Could we perhaps rename this folder to something that is more distinct from app? 

Eg. "ruter"

Reply:
I have called it $APPLICATION_ROOT now to indicate it is up to the operator where they put things. I will not rename the destinations in the manifest at this point.